app/shared/hooks/TextMessagingProvider.js

```diff
@@ -26,7 +28,7 @@ export function TextMessagingProvider({
 
   return (
     <TextMessagingContext.Provider
-      value={[textMessaging, refreshTextMessaging]}
+      value={[textMessaging, refreshTextMessaging, compliance, setCompliance]}
```

Comment:
Either this provider should have a value that's an `{}`, to be consistent w/ the return from the `useTextMessaging` hook... 

or it should be split out to two different contexts/providers so as to maintain the `[state, setState]` pattern that's familiar in React (IMHO, this is the better of the two options).